region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -971,32 +1064,50 @@
       "link": [10, 14],
       "name": "Shinespark",
       "notable": false,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 60
```

Comment:
Frame count should be lower than the hijump shinespark.  Probably the same number as the 10->9 shinespark.

Reply:
Done.